NOTES.md

```diff
@@ -1,9 +1,31 @@
-# TODO
+# Tagging system
```

Comment:
Does this file need to be committed? Are we going to start documenting "notes" here? Should we maybe just add something like this to the documentation site instead?